make/common.mk

```diff
@@ -50,7 +55,7 @@ GEN_JAVA_DIR = $(BUILD_DIR)/java
 endif
 
 ifndef J2OBJC_ARCHS
-J2OBJC_ARCHS = macosx iphone iphone64 watchv7k simulator simulator64
+J2OBJC_ARCHS = macosx iphone iphone64 watchv7k simulator simulator64 appletvos appletvsimulator
```

Comment:
We're going to accept the contribution. If we encounter any problems with having appletv builds in the distribution we can always update the J2OBJC_ARCHS variable without touching the rest of the contribution.